components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/graphql/queryanalysis/GraphqlPolicyDefinition.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *
+ *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package org.wso2.carbon.apimgt.api.model.graphql.queryanalysis;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GraphqlPolicyDefinition {
```

Comment:
shall we add a class level comment

Reply:
Not applicable anymore since the master branch has a different codebase.